src/one_dragon/base/controller/pc_screenshot/gdi_screencapper_base.py

```
@@ -0,0 +1,350 @@
+                int(height),
+                ctypes.cast(buffer, ctypes.c_void_p),
+                ctypes.byref(bmpinfo_buffer),
+                DIB_RGB_COLORS
+            )
+            if lines != int(height):
+                return None
+            img_array = self.buffer_to_ndarray(buffer, width, height)
+            screenshot = cv2.cvtColor(img_array, cv2.COLOR_BGRA2RGB)
+            if is_win_scale and standard_width and standard_height:
+                screenshot = cv2.resize(screenshot, (standard_width, standard_height))
+            return screenshot
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)